Formula/restack.rb

```diff
@@ -0,0 +1,16 @@
+class Restack < Formula
+  desc "CLI app"
+  homepage "https://github.com/restackio/homebrew-restack"
+  url "https://github.com/restackio/cli-bin/archive/refs/tags/v0.1.0.tar.gz"
```

Comment:
This packages a pre-built binary, which is not acceptable in Homebrew. https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae